jit/baseline.ts

```diff
@@ -71,7 +71,7 @@ module J2ME {
    * Emits array bounds checks. Although this is necessary for correctness, most
    * applications work without them.
    */
-  export var emitCheckArrayBounds = true;
+  export var emitCheckArrayBounds = false;
```

Comment:
I think we shouldn't do this unless we prove MIDlets work flawlessly without array bound checks.